submissions/ik-web/html-css-popup/index.html

```diff
@@ -0,0 +1,116 @@
+                        <input type="checkbox" id="popup-btn_more">
+                        <label for="popup-btn_more" class="popup-menu-btn">more</label>
+
+                        <ul class="popup-menu_more">
+
+                            <!-- Popup menu_more items -->
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/maps.png" alt="maps"><span>Maps</span></a>
+                            </li>
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/news.png" alt="news"><span>News</span></a>
+                            </li>
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/photos.png" alt="photos"><span>Photos</span></a>
+                            </li>
+
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/videos.png" alt="videos" id="videos"><span>Videos</span></a>
+                            </li>
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/tanslation.png" alt="translation"><span>Translation</span></a>
+                            </li>
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/search.png" alt="search"><span>Search</span></a>
+                            </li>
+
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/promo.png" alt="promo"><span>Promo</span></a>
+                            </li>
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/mars.png" alt="translation"><span>Mars</span></a>
+                            </li>
+                            <li>
+                                <a href="#" class="popup-menu-item"><img src="icons/collection.png" alt="collection"><span>Collection</span></a>
+                            </li>
+
+                            <!-- Popup menu_more button Kottans -->
+                            <li class="btn">
+                                <a href="https://kottans.org/"  target="_blank" class="popup-menu-btn">Even more from Kottans</a>
+                            </li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+
+            <!-- Nav icon bell -->
+            <li>
+                <a href="#" class="nav-item nav-icon"><img src="icons/bell.png" alt="bell"></a>
+            </li>
+
+            <!-- Nav user photo -->
+            <li>
+                <a href="https://github.com/ik-web" target="_blank" class="nav-item nav-icon"><img src="img/user.jpg" alt="user-photo" class="nav-user"></a>
+            </li>
+        </ul>
+    </nav>
+</body>
+</html>
```

Comment:
Add an empty line

Reply:
Should I make the last line of code empty? Did I understand correctly?

Reply:
Yeah, correctly